README.md

```diff
@@ -25,6 +25,7 @@ or install from source:
 ```
 pip install git+https://github.com/onnx/onnxmltools
 ```
+Note: If you choose to install `onnxmltools` from source code , you must set environment variable `ONNX_ML=1` before installing `onnx` dependency package for `onnxmltools`.
```

Comment:
Please remove redundant spaces, "Note:", and "dependency"

Reply:
Thanks. I note "Note:" has redundant space, but I can not find the redundant space of "dependency". Could you give me more context of this?

Reply:
If so, could you also update your PR? Thanks.

Reply:
Of course, I will update.

Reply:
@wschin I have updated. However, I find that the travis-CI's environment has some problem. CI doesn't export ONNX=='1.0.1', just export ONNX=='', which makes CI fail.